File: samples/workinprogress/chatroom/app/controllers/Global.scala
import play.api._

import actors._
import java.util.concurrent._

object MyGlobal extends GlobalSettings {
  
  override def onStart(app: Application) {
    Logger.info("Start ChatRoomActor")
  }
  
  override def onStop(app: Application) {
    Logger.info("Stop ChatRoomActor")
    ChatRoomActor.system.shutdown
  }
  
}

File: documentation/manual/working/javaGuide/main/application/code/javaguide/application/startstop/Global.java
/*
 * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
 */
package javaguide.application.startstop;

//#global
import play.*;

public class Global extends GlobalSettings {

    public void onStart(Application app) {
        Logger.info("Application has started");
    }

    public void onStop(Application app) {
        Logger.info("Application shutdown...");
    }

}
//#global
